sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClientOptions.cs

```diff
@@ -86,10 +86,15 @@ public TimeSpan ConnectionIdleTimeout
         }
 
         /// <summary>
-        /// The set of options to use for determining whether a failed operation should be retried and,
+        /// The set of options to use for determining whether a failed service operation should be retried and,
         /// if so, the amount of time to wait between retry attempts.  These options also control the
-        /// amount of time allowed for receiving messages and other interactions with the Service Bus service.
+        /// amount of time allowed for the individual network operations used for interactions with the Service Bus service.
         /// </summary>
+        /// <remarks>
+        /// The retry options are only considered for interactions with the Service Bus service. They do not apply to failures when an
+        /// application is processing messages.  Developers are responsible for error handling and retries for failures that occur during
```

Comment:
I think this is referring specifically to when using the processor? Are we essentially saying that failures in the user event handler will not be governed by the retry policy? If so, it might be better to mention that explicitly.

Reply:
Yeah, I was trying to keep it generic, but I guess that it really is scoped to the processor since pulling from the receiver has a pretty clear boundary.

Reply:
Yep, putting the caveat in the Remarks sections makes the most sense.